EntityFramework.Utilities/EntityFramework.Utilities/SqlQueryProvider.cs

```diff
@@ -56,7 +56,9 @@ public void InsertItems<T>(IEnumerable<T> items, string schema, string tableName
                 {
                     con.Open();
                 }
-                using (SqlBulkCopy copy = new SqlBulkCopy(con))
+                using (var copy = transaction == null
+                    ? new SqlBulkCopy(con.ConnectionString, copyOptions)
```

Comment:
Overloads for `SqlBulkCopy` are a bit weird, I have to use a string connection here.